app/controllers/courses_controller.rb

```diff
@@ -237,6 +239,32 @@ def wiki_params
       .permit(:language, :project)
   end
 
+  def courses_wikis_params
+    params
+      .require(:course)
+      .permit(wikis: [:language, :project])
+  end
+
+  def update_courses_wikis
+    @course.ensure_home_wiki_in_courses_wikis
```

Comment:
This should not be necessary, as this method is only called immediately after a course record was saved anyway, and now we have that `before_save` hook.